scriptlets/install_checkbox_snaps.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+# Installs Checkbox and exports the installed version on the DUT
+# to: CHECKBOX_VERSION
+
+if [[ "$#" != "4" ]]; then
+  echo "Usage: $(basename ${BASH_SOURCE[0]}) risk runtime_name frontend_name frontend_track"
+  exit 1
+fi
+
+echo "Installing Checkbox snaps"
+
+RISK=$1
+shift
+RUNTIME_NAME=$1
+shift
+FRONTEND_NAME=$1
+shift
+FRONTEND_TRACK=$1
+shift
```

Comment:
Are there reasonable default values for any of these variables? If so, I vote for making them optional. I know it will probably make this script longer in order to parse them but it will also make the calls to the script more explicit.

Reply:
No, I want all of these to be explicit, I expect this to be called like this
```
install_checkbox_snaps beta checkbox22 checkbox 22.04
```